R/emmeans-tidiers.R

```diff
@@ -126,9 +126,43 @@ tidy.emmGrid <- function(x, ...) {
   tidy_emmeans(x, ...)
 }
 
+#' @templateVar class emmGrid
+#' @template title_desc_tidy
+#' 
+#' @param x An `summary_emm` object.
+#' @inherit tidy.lsmobj params examples details 
+#'   
```

Comment:
It would be good to include the `joint_test()` example from the PR thread here in the documentation

Reply:
Makes sense, I'll get right on it.